### ISSUE TITLE
[IOS-XE] Parser for "show interfaces" does not accommodate pseudowires or ucse interfaces.

### ISSUE THREAD

**Issue by ChristopherJHart** [CiscoTestAutomation/genieparser#876]
If the output of `show interfaces` command contains any of the following output, the Genie parser will not parse those interfaces:
```
ucse1/0/1 is administratively down, line protocol is down 
  Hardware is UCS-E160S-M3/K9, address is 0045.1d12.d8b1 (bia 0045.1d12.d8b1)
  MTU 1500 bytes, BW 1000000 Kbit/sec, DLY 10 usec, 
     reliability 255/255, txload 1/255, rxload 1/255
  Encapsulation ARPA, loopback not set
  Keepalive not supported 
  Full Duplex, 1000Mbps, link type is force-up, media type is Internal
  output flow-control is unsupported, input flow-control is unsupported
  ARP type: ARPA, ARP Timeout 04:00:00
  Last input never, output never, output hang never
  Last clearing of "show interface" counters never
  Input queue: 0/375/0/0 (size/max/drops/flushes); Total output drops: 0
  Queueing strategy: fifo
  Output queue: 0/40 (size/max)
  5 minute input rate 0 bits/sec, 0 packets/sec
  5 minute output rate 0 bits/sec, 0 packets/sec
     0 packets input, 0 bytes, 0 no buffer
     Received 0 broadcasts (0 IP multicasts)
     0 runts, 0 giants, 0 throttles 
     0 input errors, 0 CRC, 0 frame, 0 overrun, 0 ignored
     0 watchdog, 0 multicast, 0 pause input
     0 packets output, 0 bytes, 0 underruns
     Output 0 broadcasts (0 IP multicasts)
     0 output errors, 0 collisions, 1 interface resets
     0 unknown protocol drops
     0 babbles, 0 late collision, 0 deferred
     0 lost carrier, 0 no carrier, 0 pause output
     0 output buffer failures, 0 output buffers swapped out
     
pseudowire1 is up
    Description: pseudowire to core router
    MTU 9198 bytes, BW not configured
    Encapsulation mpls
    Peer IP 192.0.2.3, VC ID 1
    RX
      0 packets 0 bytes 0 drops
    TX
      0 packets 0 bytes 0 drops
```

**Comment by Neeraja1bala** [comment]
Hello @ChristopherJHart 

Thanks for reaching out us, kindly give some time i will check and let you know.

**Comment by SohanTirpude** [comment]
Hello @ChristopherJHart,

The PR has got merged and the fix will be available in the pyats 24.9 release. Meanwhile, I am closing this ticket. If you are facing any other issues, then kindly open a new ticket.

Thank you.
